websocket_push_session.go

```diff
@@ -239,10 +290,39 @@ func (s *wsPushSession) listen() {
 					zap.String("sessionID", s.id),
 					zap.String("pushConfig", pushConfig.String()),
 				)
-				s.close(websocket.CloseUnsupportedData)
-				return
+
+				if s.handlesErrorEvents() {
+					s.setErrorState(true)
+					msgpack, json, err := prepareEventData(elemental.NewErrorEvent(elemental.Error{
+						Title:       "Bad request",
+						Subject:     "bahamut",
+						Description: fmt.Sprintf("unable to parse identity filters: %s", err),
+						Data: map[string]interface{}{
+							"attribute": "filters",
```

Comment:
I don't think we want to use the `attribute` key here. This is not a Gaia model attribute. We should use another key

Reply:
I couldn't think of something better 😅 

Open to suggestions!

Reply:
Discussed offline: we will rename the key to `pushconfig` instead

Reply:
settling down on `pushconfig`

Reply:
Donezo ✅